shortfin/README.md

```diff
@@ -1,20 +1,44 @@
 # shortfin - SHARK inference library and serving engine
 
-## Simple User Installation
+The shortfin project is SHARK's open source, high performance inference library
+and serving engine. Shortfin consists of these major components:
 
-Install:
+* The "libshortfin" inference library written in C/C++ and built on
+  [IREE](https://github.com/iree-org/iree)
+* Python bindings for the underlying inference library
+* Example applications in
+  ['shortfin_apps'](https://github.com/nod-ai/SHARK-Platform/tree/main/shortfin/python/shortfin_apps)
+  built using the python bindings
 
+## Prerequisites
+
+* Python 3.11+
```

Comment:
Unit tests pass with 3.10 and we build wheels for 3.10. However, we should probably test the apps for which we say that those are supported.

Reply:
I heard someone say 3.11+ this morning, so starting there. We can expand when we have better test / QA coverage.